src/utils/renderHtml/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import parse, {
+  Element,
+  HTMLReactParserOptions,
+  domToReact,
+} from "html-react-parser";
+
+/**
+ * renderHtml takes a string and converts it into an array of domnodes and
```

Comment:
I'm curious if you looked into [`react-highlight-words`](https://github.com/bvaughn/react-highlight-words)? It's also written by bvaughn (he's everywhere... 🤔). It could be a simpler approach since we're only using this for highlighting words

Reply:
I have! But this actually does more than just highlight the words it also handles rendering the ansii transformed html string into dom elements that are generated by ansii up. I figured might as well just extend the same library we are using elsewhere to also do the syntax highlighting instead of pulling in yet another library.